modules/shared/src/main/resources/logback-included.xml

```diff
@@ -6,13 +6,17 @@
     <property name="HTTP_ROLLING_LOG" value="logs/archived/http.%d{yyyy-MM-dd}.%i.log.gz"/>
     <property name="GOSSIP_LOG" value="logs/gossip.log"/>
     <property name="GOSSIP_ROLLING_LOG" value="logs/archived/gossip.%d{yyyy-MM-dd}.%i.log.gz"/>
+    <property name="PEER_SELECT_LOG" value="logs/peerselect.log"/>
```

Comment:
can you name the log file either `peer_select.log` or `peer-select.log`?

Reply:
The log files have been renamed `peer-select`.